dashboard/lib/pd/payment/payment_factory.rb

```diff
@@ -27,7 +27,7 @@ def self.get_calculator_class(workshop)
         ].include?(workshop.course)
       end
 
-      nil
+      PaymentCalculatorUnpaid
```

Comment:
Yes, for 2 specific examples: https://github.com/code-dot-org/code-dot-org/pull/12651/files#diff-2b6b6c2c760761d8b2e4668eae8dc74eR56

I could add a test with a mocked made up course we know will never match to verify the fallback.